test/StreamProviderTests/Integration/OrleansTests.cs

```diff
@@ -33,198 +33,203 @@ public class OrleansTests
         private static CancellationTokenSource GetTokenSource(TimeSpan? time = null) => new CancellationTokenSource(time ?? DefaultBlockingTimeout);
 
         [Fact]
-        public Task RedisStreamCanSendAndReceiveItem()
+        public async Task RedisStreamCanSendAndReceiveItem()
         {
-            var clusterFixture = new StreamingClusterFixture();
-
-            return clusterFixture.Dispatch(async () =>
+            using (var clusterFixture = new StreamingClusterFixture())
```

Comment:
Was never disposing the cluster fixture. Seems it wasn't an issue pre-2.1.0 but it definitely is post.